requirements.txt

```diff
@@ -1,6 +1,6 @@
 ansible==6.0.0
 openstacksdk
-python-openstackclient
+python-openstackclient==6.6.1 # v7.0.0 has a bug re. rebuild
```

Comment:
Not important for here, but someone smarter than me once told me to pin versions like this:
```
python-openstackclient<7.0.0
```

because it makes complex environments easier for pip to solve...

Reply:
good to know, thanks. Looks like the bug report has some traction so I guess we will be changing this soon anyway.